drivers/modem/Kconfig.hl7800

```diff
@@ -288,8 +288,15 @@ config MODEM_HL7800_BOOT_IN_AIRPLANE_MODE
 
 endchoice
 
-config GET_IP_ADDR_INFO_ATTEMPTS
+config MODEM_HL7800_GET_IP_ADDR_INFO_ATTEMPTS
 	int "Number of attempts to get IP address info during init"
 	default 4
 
+config MODEM_HL7800_GPS
+	bool "Enable GPS command and handlers"
```

Comment:
Maybe we should use GNSS instead of GPS as a more general term?

Adding GNSS should be really a separate PR. The thing is that I would much prefer to establish a generic API for GNSS providers, like proposed in #30564. However to achieve that, we first need to have list of initial GNSS modules we want to handle with proposed API to make sure we start with something scalable.